File: weekOfOct2/numIdenticalPairs.js
/**
 * @param {url} https://leetcode.com/problems/number-of-good-pairs/submissions/?envType=daily-question&envId=2023-10-03
 * @param {number[]} nums
 * @return {number}
 */
var numIdenticalPairs = function(nums) {
  let pairs = 0;

  for(let i = 0; i < nums.length; i++) {
      for(let j = i + 1; j < nums.length; j++) {
          if(nums[i] == nums[j]) pairs++;
      }
  }

  return pairs
};

File: weekOfOct2/numIdenticalPairs.py
class Solution(object):
    def numIdenticalPairs(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """

        print(self)

        pairs = 0

        for i in range (0, len(nums)):
            for j in range (i + 1, len(nums)):
                if nums[i] == nums[j]:
                    pairs += 1

        return pairs